consensus/src/runway/collection.rs

```diff
@@ -236,6 +237,11 @@ impl<'a, H: Hasher, D: Data, MK: KeyBox> IO<'a, H, D, MK> {
         {
             warn!(target: "AlephBFT-runway", "unable to send resolved request:  {}", e);
         }
+        info!(target: "AlephBFT-runway", "Finished initial unit collection with status: {:?}", self.collection.status());
+    }
+
+    fn status_report(&self) {
+        info!(target: "AlephBFT-runway", "Initial unit collection status report: {:?}", self.collection.status());
```

Comment:
The rationale here is as follows: we want this logged in all production nodes (ours and then also external), because it's crucial diagnostic information. So how to achieve it? I think the only reasonable way is to log it as `info`. If this is debug, and one must add special flags to enable this log then I'm 100% sure that when the time comes and the logs will be needed, then by some strange coincidence the logs will turn out to be disabled.